instrumentation/opentelemetry-instrumentation-psycopg2/src/opentelemetry/instrumentation/psycopg2/package.py

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 
-_instruments = ("psycopg2 >= 2.7.3.1",)
+_instruments = (("psycopg2 >= 2.7.3.1", "psycopg2-binary >= 2.7.3.1"),)
```

Comment:
```suggestion
_instruments = ("psycopg2 >= 2.7.3.1", "psycopg2-binary >= 2.7.3.1")
```
This is what the kafka-python is doing, please take a look at opentelemetry-instrumentation-kafka-python/pyproject.toml, with that updated you should be able to drop the dependencies,py changes.

Reply:
I've done the minimal implementation here https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1422